scripts/scripts.js

```diff
@@ -12,24 +12,114 @@
+    ee: { ietf: 'et-EE', tk: 'aaz7dvd.css' },
+    es: { ietf: 'es-ES', tk: 'oln4yqj.css' },
+    fr: { ietf: 'fr-FR', tk: 'vrk5vyv.css' },
+    gr_en: { ietf: 'en-GR', tk: 'pps7abe.css' },
+    ie: { ietf: 'en-GB', tk: 'pps7abe.css' },
+    il_en: { ietf: 'en-IL', tk: 'pps7abe.css' },
+    it: { ietf: 'it-IT', tk: 'bbf5pok.css' },
+    lv: { ietf: 'lv-LV', tk: 'aaz7dvd.css' },
+    lt: { ietf: 'lt-LT', tk: 'aaz7dvd.css' },
+    lu_de: { ietf: 'de-LU', tk: 'vin7zsi.css' },
+    lu_en: { ietf: 'en-LU', tk: 'pps7abe.css' },
+    lu_fr: { ietf: 'fr-LU', tk: 'vrk5vyv.css' },
+    hu: { ietf: 'hu-HU', tk: 'aaz7dvd.css' },
+    mt: { ietf: 'en-MT', tk: 'pps7abe.css' },
+    mena_en: { ietf: 'en', tk: 'pps7abe.css' },
+    nl: { ietf: 'nl-NL', tk: 'cya6bri.css' },
+    no: { ietf: 'no-NO', tk: 'aaz7dvd.css' },
+    pl: { ietf: 'pl-PL', tk: 'aaz7dvd.css' },
+    pt: { ietf: 'pt-PT', tk: 'inq1xob.css' },
+    ro: { ietf: 'ro-RO', tk: 'aaz7dvd.css' },
+    sa_en: { ietf: 'en', tk: 'pps7abe.css' },
+    ch_de: { ietf: 'de-CH', tk: 'vin7zsi.css' },
+    si: { ietf: 'sl-SI', tk: 'aaz7dvd.css' },
+    sk: { ietf: 'sk-SK', tk: 'aaz7dvd.css' },
+    ch_fr: { ietf: 'fr-CH', tk: 'vrk5vyv.css' },
+    fi: { ietf: 'fi-FI', tk: 'aaz7dvd.css' },
+    se: { ietf: 'sv-SE', tk: 'fpk1pcd.css' },
+    ch_it: { ietf: 'it-CH', tk: 'bbf5pok.css' },
+    tr: { ietf: 'tr-TR', tk: 'aaz7dvd.css' },
+    ae_en: { ietf: 'en', tk: 'pps7abe.css' },
+    uk: { ietf: 'en-GB', tk: 'pps7abe.css' },
+    at: { ietf: 'de-AT', tk: 'vin7zsi.css' },
+    cz: { ietf: 'cs-CZ', tk: 'aaz7dvd.css' },
+    bg: { ietf: 'bg-BG', tk: 'aaz7dvd.css' },
+    ru: { ietf: 'ru-RU', tk: 'aaz7dvd.css' },
+    ua: { ietf: 'uk-UA', tk: 'aaz7dvd.css' },
+    il_he: { ietf: 'he', tk: 'nwq1mna.css' },
+    ae_ar: { ietf: 'ar', tk: 'nwq1mna.css' },
+    mena_ar: { ietf: 'ar', tk: 'dis2dpj.css' },
+    sa_ar: { ietf: 'ar', tk: 'nwq1mna.css' },
+    // Asia Pacific
+    au: { ietf: 'en-AU', tk: 'pps7abe.css' },
+    hk_en: { ietf: 'en-HK', tk: 'pps7abe.css' },
+    in: { ietf: 'en-GB', tk: 'pps7abe.css' },
+    id_id: { ietf: 'id', tk: 'czc0mun.css' },
+    id_en: { ietf: 'en', tk: 'pps7abe.css' },
+    my_ms: { ietf: 'ms', tk: 'sxj4tvo.css' },
+    my_en: { ietf: 'en-GB', tk: 'pps7abe.css' },
+    nz: { ietf: 'en-GB', tk: 'pps7abe.css' },
+    ph_en: { ietf: 'en', tk: 'pps7abe.css' },
+    ph_fil: { ietf: 'fil-PH', tk: 'ict8rmp.css' },
+    sg: { ietf: 'en-SG', tk: 'pps7abe.css' },
+    th_en: { ietf: 'en', tk: 'pps7abe.css' },
+    in_hi: { ietf: 'hi', tk: 'aaa8deh.css' },
+    th_th: { ietf: 'th', tk: 'aaz7dvd.css' },
+    cn: { ietf: 'zh-CN', tk: 'puu3xkp' },
+    hk_zh: { ietf: 'zh-HK', tk: 'jay0ecd' },
+    tw: { ietf: 'zh-TW', tk: 'jay0ecd' },
+    jp: { ietf: 'ja-JP', tk: 'dvg6awq' },
+    kr: { ietf: 'ko-KR', tk: 'qjs5sfm' },
+    // Langstore Support.
+    langstore: { ietf: 'en-US', tk: 'hah7vzn.css' },
   },
+  // geoRouting: 'on',
+  productionDomain: 'business.adobe.com',
```

Comment:
Is this the correct domain?

Reply:
It is what the sharepoint is pointing to at the moment, but I will ask Malik and Dave. Preview of the documents on sharepoint is not working atm.